src/dbnode/storage/bootstrap/process.go

```diff
@@ -340,8 +347,7 @@ func (b bootstrapProcess) targetRanges(
 	// bootstrap with persistence so we don't keep the full raw
 	// data in process until we finish bootstrapping which could
 	// cause the process to OOM.
-	return []TargetRange{
-		{
+	return TargetRange{
```

Comment:
nit: just to make this a bit clearer, can you do something like
```golang
dataRange := TargetRange{...}
indexRange := TargetRange{...}
return dataRange, indexRange
```

Reply:
Sure yeah